tikv-migration/tikv-migration-verify-pipeline.yaml

```diff
@@ -29,3 +29,21 @@ tasks:
       credentials:
         - jenkinsID: codecov-token-tikv-migration
           key: TIKV_MIGRATION_CODECOV_TOKEN
+
+    - taskName: "cdc-it" 
```

Comment:
Suggest to use the name `cdc-integration-test`. `it` is not a well known abbreviation.